docs/install_guide_ova.md

```diff
@@ -0,0 +1,114 @@
+    ![Screenshot of customize harbor](img/ovainstall/customizeharbor.png)
+    If the Authentication Mode is set to ldap_auth, need to config Harbor LDAP settings:
+    - LDAP Base DN: The base DN to lookup user
+    - LDAP UID: The attribute field to match user, such as uid, cn, email or other attributes.
+    - LDAP URL: The url for LDAP endpoint
+    - Search DN: The user's DN who has the permission to search the LDAP Server
+    - Search DN Password: The password of search DN.
+    - Search Filter: The filter to search users, can be empty.
+    - Search Scope: The scope to search users.
+    ![Screenshot of customize ldap](img/ovainstall/customizeldap.png)
+
+5. On the **Customize template** page, under **System**, set the root password for the appliance VM and the option for **Permit Root Login**. 
+
+    Setting the root password for the appliance is mandatory. 
+
+    **IMPORTANT**: You require SSH access to the vSphere Integrated Containers appliance to perform upgrades. You can also use SSH access in exceptional cases that you cannot handle through standard remote management or CLI tools. Only use SSH to access the appliance when instructed to do so in the documentation.
+    ![Screenshot of customize template system](img/ovainstall/system.png)
+
+6. Expand **Networking Properties** and optionally configure a static IP address for the appliance VM. 
+
+    To use DHCP, leave the **Networking Properties** blank.
+
+    **IMPORTANT**: If you set a static IP address for the appliance, use spaces to separate DNS servers. Do not use comma separation for DNS servers. 
+
+    - Leave the text boxes blank to use auto-generated certificates.
+   
+7. When the deployment completes, refresh the current page and power on the appliance VM.
+   ![Screenshot of power on](img/ovainstall/poweron.png)
+   Press Enter to login
+   ![Screenshot of login root](img/ovainstall/poweron2.png)
+
+   Login to the console with root username and password, and get the harbor appliance address by ifconfig.
+
+   If you deployed the appliance so that it obtains its address via DHCP, go to the **Summary** tab for the appliance VM and note the address.
+
+8. (Optional) If you provided a static network configuration, view the network status of the appliance.
+
+    1. In the **Summary** tab for the appliance VM, launch the VM console
+    2. In the VM console, press the right arrow key. 
+
+    The network status shows whether the network settings that you provided during the deployment match the settings with which the appliance is running. If there are mismatches, power off the appliance and select **Edit Settings** > **vApp Options** to correct the network settings.
+    
+9. In a browser, go to  https://<i>harbor_appliance_address</i> and when prompted, enter the username admin and the password of admin input in step 4
```

Comment:
default is the FQDN name. do you mean the harbor_appliance_address is the FQDN?

Reply:
Change to DNS Name